android/app/build.gradle

```diff
@@ -135,6 +135,8 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-push-notification')
+    compile project(':react-native-push-notifications')
```

Comment:
is this correct?

Reply:
The readme says on android you have to link manually and I haven't worked on the android part at all yet. That's also the reason why the android build is broken.